operations/kubernetes/encryption/README.md

```diff
@@ -231,3 +241,40 @@ If necessary, a forced rewrite of secret data can be performed. Generally unnece
     Waiting for daemon set "cray-k8s-encryption" rollout to finish: 2 of 3 updated pods are available...
     daemon set "cray-k8s-encryption" successfully rolled out
     ```
+
+## Key rotation
+
+Rotation of the encryption keys is similar to disabling encryption. The first cipher passed to `encryption.sh` is the cipher that will be used to encrypt secrets.
+
+All subsequent ciphers provided allow for any secrets encrypted with that cipher to be read but will not be used for new secrets.
+
+* (`ncn-m#`) The `encryption.sh` script needs to be run with the new key as the first switch and the existing key as a subsequent switch.
+
+    As shown in the following command example, always run `encryption.sh` with a leading space on the command line. This will cause Bash to not record the command in the `.bash_history` file.
```

Comment:
```suggestion
    **`NOTE`**: As shown in the following command example, always run `encryption.sh` with a leading space on the command line. This will cause Bash to not record the command in the `.bash_history` file.
```
This seems like it is important and might be overlooked by the reader when they run the command.

Reply:
Copy/pasted from the other lines, I can update all of them at once?